tests/app/clients/test_document_download.py

```diff
@@ -33,26 +32,27 @@ def test_upload_document(document_download):
     assert resp == 'https://document-download/services/service-id/documents/uploaded-url'
 
 
-def test_should_raise_for_status(document_download):
+def test_should_raise_400s_as_DocumentDownloadErrors(document_download):
     with pytest.raises(DocumentDownloadError) as excinfo, requests_mock.Mocker() as request_mock:
         request_mock.post('https://document-download/services/service-id/documents', json={
-            'error': 'Invalid encoding'
-        }, status_code=403)
+            'error': 'Invalid mime type'
+        }, status_code=400)
 
         document_download.upload_document('service-id', 'abababab')
 
-    assert excinfo.value.message == 'Invalid encoding'
-    assert excinfo.value.status_code == 403
+    assert excinfo.value.message == 'Invalid mime type'
+    assert excinfo.value.status_code == 400
 
 
-def test_should_raise_for_connection_errors(document_download):
-    with pytest.raises(DocumentDownloadError) as excinfo, requests_mock.Mocker() as request_mock:
+def test_should_raise_other_statuses_as_exceptions(document_download):
```

Comment:
Optional:

Could rename test

`test_should_raise_non_400_statuses_as_exceptions` which is a bit more explicit to what is being tested.

I don't know if we expect any other response codes from doc download, if so you could parametrize them

Reply:
i had a look, we don't return any other user errors, just two 400s (file has a virus or file isn't of a recognised type)